[2023-06-29T11:32:17.737+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select permission to 509
[2023-06-29T11:32:17.739+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select'
[2023-06-29T11:32:17.773+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select permission to 509
[2023-06-29T11:32:17.774+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select'
[2023-06-29T11:32:17.817+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-29T11:20:00+00:00 [queued]>
[2023-06-29T11:32:17.832+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-29T11:20:00+00:00 [queued]>
[2023-06-29T11:32:17.833+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-29T11:32:17.852+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_select> on 2023-06-29 11:20:00+00:00
[2023-06-29T11:32:17.859+0000] {standard_task_runner.py:57} INFO - Started process 7949 to run task
[2023-06-29T11:32:17.862+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Loop', 'create_select', 'scheduled__2023-06-29T11:20:00+00:00', '--job-id', '726', '--raw', '--subdir', 'DAGS_FOLDER/test_loop.py', '--cfg-path', '/tmp/tmpl_vwdgge']
[2023-06-29T11:32:17.863+0000] {standard_task_runner.py:85} INFO - Job 726: Subtask create_select
[2023-06-29T11:32:17.920+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select permission to 509
[2023-06-29T11:32:17.921+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-29T11:20:00+00:00/task_id=create_select'
[2023-06-29T11:32:17.924+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-29T11:20:00+00:00 [running]> on host 4f3859432e6c
[2023-06-29T11:32:18.100+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Loop' AIRFLOW_CTX_TASK_ID='create_select' AIRFLOW_CTX_EXECUTION_DATE='2023-06-29T11:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-29T11:20:00+00:00'
[2023-06-29T11:32:18.102+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-29T11:32:18.103+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run --model logs_enhanced']
[2023-06-29T11:32:18.118+0000] {subprocess.py:86} INFO - Output:
[2023-06-29T11:32:22.656+0000] {subprocess.py:93} INFO - [0m11:32:22  Running with dbt=1.4.6
[2023-06-29T11:32:29.384+0000] {subprocess.py:93} INFO - [0m11:32:29  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-29T11:32:29.385+0000] {subprocess.py:93} INFO - There are 2 unused configuration paths:
[2023-06-29T11:32:29.386+0000] {subprocess.py:93} INFO - - models.badge_5.toto_select
[2023-06-29T11:32:29.387+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day_lu
[2023-06-29T11:32:29.429+0000] {subprocess.py:93} INFO - [0m11:32:29  Found 2 models, 0 tests, 0 snapshots, 0 analyses, 442 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics
[2023-06-29T11:32:29.433+0000] {subprocess.py:93} INFO - [0m11:32:29
[2023-06-29T11:32:31.433+0000] {subprocess.py:93} INFO - [0m11:32:31  Concurrency: 200 threads (target='dev')
[2023-06-29T11:32:31.434+0000] {subprocess.py:93} INFO - [0m11:32:31
[2023-06-29T11:32:31.487+0000] {subprocess.py:93} INFO - [0m11:32:31  1 of 1 START sql table model ***.logs_enhanced ............................. [RUN]
[2023-06-29T11:32:36.309+0000] {subprocess.py:93} INFO - [0m11:32:36  1 of 1 OK created sql table model ***.logs_enhanced ........................ [[32mSUCCESS 1[0m in 4.82s]
[2023-06-29T11:32:36.379+0000] {subprocess.py:93} INFO - [0m11:32:36
[2023-06-29T11:32:36.380+0000] {subprocess.py:93} INFO - [0m11:32:36  Finished running 1 table model in 0 hours 0 minutes and 6.94 seconds (6.94s).
[2023-06-29T11:32:36.418+0000] {subprocess.py:93} INFO - [0m11:32:36
[2023-06-29T11:32:36.419+0000] {subprocess.py:93} INFO - [0m11:32:36  [32mCompleted successfully[0m
[2023-06-29T11:32:36.421+0000] {subprocess.py:93} INFO - [0m11:32:36
[2023-06-29T11:32:36.423+0000] {subprocess.py:93} INFO - [0m11:32:36  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2023-06-29T11:32:37.325+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-29T11:32:37.359+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Loop, task_id=create_select, execution_date=20230629T112000, start_date=20230629T113217, end_date=20230629T113237
[2023-06-29T11:32:37.418+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-29T11:32:37.440+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
